doc/operators/start_network.rst

```diff
@@ -35,7 +35,7 @@ When starting up, the node generates its own key pair and outputs the certificat
 
 .. note:: The network certificate should be distributed to users and members to be used as the certificate authority (CA) when establishing a TLS connection with any of the nodes part of the CCF network. When using curl, this is passed as the ``--cacert`` argument.
 
-The certificates and recovery public keys of initial members of the consortium are specified via ``--member-info``. For example, if 3 members should be added to CCF, operators should specify ``--member-info member1_cert.pem,member1_enc_pubk.pem``, ``--member-info member2_cert.pem,member2_enc_pubk.pem``, ``--member-info member3_cert.pem,member3_enc_pubk.pem``.
+The certificates, encryption public keys and member data of initial members of the consortium are specified via ``--member-info``. For example, if 3 members (one recovery member with member data, one recovery member without member data and one non-recovery member with member data) should initially be added to CCF, operators should specify ``--member-info member1_cert.pem,member1_enc_pubk.pem,member1_data.json``, ``--member-info member2_cert.pem,member2_enc_pubk.pem``, ``--member-info member3_cert.pem,,member2_data.json`` (note the empty public encryption key path "``,,``" for the third member).
```

Comment:
I think this example is now covering too many things, and they should be separately listed. Something like:

> For example:
> - A recovery member with member data
> `--member-info member1_cert.pem,member1_enc_pubk.pem,member1_data.json`
> - A recovery member without member data
> `--member-info member2_cert.pem,member2_enc_pubk.pem`
> - A non-recovery member with member data (note the empty public encryption key path, indicating a non-recovery member)
> `--member-info member3_cert.pem,,member2_data.json`

Reply:
Good point. I've also added a bullet point for non-recovery, no member data member, i.e. `--member-info member_cert.pem`.